CMakeLists.txt

```diff
@@ -103,6 +110,14 @@ target_include_directories(onnxruntime-genai-static PRIVATE ${onnxruntime_extens
 target_include_directories(onnxruntime-genai-static PUBLIC ${onnxruntime_extensions_SOURCE_DIR}/shared/api/)
 target_link_libraries(onnxruntime-genai PRIVATE onnxruntime_extensions)
 target_link_libraries(onnxruntime-genai-static PUBLIC onnxruntime_extensions)
+target_include_directories(onnxruntime-genai PUBLIC ${llguidance_SOURCE_DIR}/parser/)
+target_include_directories(onnxruntime-genai-static PUBLIC ${llguidance_SOURCE_DIR}/parser/)
+target_link_libraries(onnxruntime-genai PRIVATE llguidance_parser)
+target_link_libraries(onnxruntime-genai-static PUBLIC llguidance_parser)
+if (WIN32)
+  target_link_libraries(onnxruntime-genai PRIVATE bcrypt)
```

Comment:
Why this only needed on WIN32? Maybe add some comments.

Reply:
rust std lib depends on it https://github.com/corrosion-rs/corrosion/issues/119. I'll add comment later